packages/runtime/id-compressor/src/idCompressor.ts

```diff
@@ -3,14 +3,35 @@
  * Licensed under the MIT License.
  */
 
-import { assert } from "@fluidframework/core-utils";
 import { bufferToString, stringToBuffer } from "@fluid-internal/client-utils";
 import { ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
+import { assert } from "@fluidframework/core-utils";
 import { ITelemetryLoggerExt, createChildLogger } from "@fluidframework/telemetry-utils";
+import { FinalSpace } from "./finalSpace.js";
+import { FinalCompressedId, LocalCompressedId, NumericUuid, isFinalId } from "./identifiers.js";
+import {
+	Index,
+	readBoolean,
+	readNumber,
+	readNumericUuid,
+	writeBoolean,
+	writeNumber,
+	writeNumericUuid,
+} from "./persistanceUtilities.js";
+import { SessionSpaceNormalizer } from "./sessionSpaceNormalizer.js";
+import {
+	IdCluster,
+	Session,
+	Sessions,
+	getAlignedFinal,
+	getAlignedLocal,
+	lastFinalizedFinal,
+	lastFinalizedLocal,
+} from "./sessions.js";
 import {
-	IdCreationRange,
 	IIdCompressor,
 	IIdCompressorCore,
+	IdCreationRange,
```

Comment:
This seems like unfortunate ordering. I would never have put all caps before all lowercase letters.